### PR TITLE
Random jpeg quality preprocessing layer

### PR DESCRIPTION
**Pull request by artu1999**
See #135. Applies random jpeg compression artifcats to an image to induce noise.

### CODE REVIEW

File: keras_cv/layers/__init__.py [hunk 1]
```diff
@@ -25,9 +25,6 @@
 from tensorflow.keras.layers import Rescaling
 from tensorflow.keras.layers import Resizing
 
-from keras_cv.layers.preprocessing.auto_contrast import AutoContrast
-from keras_cv.layers.preprocessing.channel_shuffle import ChannelShuffle
```
Comment (LukeWood) [inline keras_cv/layers/__init__.py:29]: revert spurious changes

File: keras_cv/layers/preprocessing/gaussian_blur.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+import tensorflow_addons as tfa
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+
+
+class GaussianBlur(layers.Layer):
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/gaussian_blur.py:20]: Looks like you need to rebase on origin/master

```
git fetch origin
git rebase origin/master -x theirs
```

**Review by LukeWood — CHANGES_REQUESTED** [review]

**Comment by LukeWood** [comment]
@artu1999 thanks for the PR!  Please rebase on the latest origin/master!

**Comment by Ashwin-Ravi** [comment]
@artu1999 Following up to see if you would be returning to the PR?
I could take over, in case you would not be available to work on this.

**Comment by artu1999** [comment]
> @artu1999 Following up to see if you would be returning to the PR? I could take over, in case you would not able to return to this.

@Ashwin-Ravi @LukeWood I had a couple of hectic weeks with my dissertation lately, but I'm submitting it in two days, I will return on the PR straight after that!

**Comment by LukeWood** [comment]
> > @artu1999 Following up to see if you would be returning to the PR? I could take over, in case you would not able to return to this.
> 
> @Ashwin-Ravi @LukeWood I had a couple of hectic weeks with my dissertation lately, but I'm submitting it in two days, I will return on the PR straight after that!

awesome, thanks for letting us know @artu1999.  Good luck on your dissertation!

**Comment by LukeWood** [comment]
Closing in favor of your newer PR.

**Pull request closed without merging**